tests/test_models.py

```diff
@@ -1739,55 +1738,6 @@ def test_string_conversion(self, method):
         assert expected_to_str == method(dsc)
 
 
-@ddt.ddt
-@mark.django_db
-class TestEnterpriseCustomerPluginConfiguration(unittest.TestCase):
```

Comment:
These tests are for `EnterpriseCustomerPluginConfiguration` abstract model. Django has removed abstract model instantiating with this error `Instantiating an abstract model now raises TypeError.` in [django3.2](https://docs.djangoproject.com/en/3.2/releases/3.2/#miscellaneous). Django [PR](https://github.com/django/django/pull/13267) in which it is implemented.

So no need to test this functionality here.